ompi/mca/osc/ucx/osc_ucx_component.c

```diff
@@ -356,7 +356,7 @@ static int component_select(struct ompi_win_t *win, void **base, size_t size, in
         ret = opal_progress_register(progress_callback);
         if (OMPI_SUCCESS != ret) {
             OSC_UCX_VERBOSE(1, "opal_progress_register failed: %d", ret);
-            goto error;
+            goto select_unlock;
```

Comment:
could just remove this line, you would end up in `select_unlock` anyway

Reply:
@brminich :) I thought about doing it, but then I thought, what if code for whatever reason is added in between, this error path may go unchecked. Maybe for clarity I prefer to leave it unless there are strong feelings to remove it and rely on fallthrough.

Reply:
ok